zulipterminal/model.py

```diff
@@ -1243,6 +1243,13 @@ def get_subscription_color(self, stream_id: int) -> Optional[str]:
             return self._unsubscribed_streams[stream_id]["color"]
         return None
 
+    def get_subscription_email(self, stream_id: int) -> Optional[str]:
+        if stream_id in self.stream_dict:
+            return self.stream_dict[stream_id]["email_address"]
+        elif stream_id in self._unsubscribed_streams:
+            return self._unsubscribed_streams[stream_id]["email_address"]
+        return None
```

Comment:
Since this method is only being used in the `StreamInfoView` and you haven't added the feature of viewing stream info for a never subscribed stream, can the base condition of returning `None` be avoided? The return value of this method is currently attached to a copy command, and having to copy a `None` to the clipboard doesn't seem appropriate.

Reply:
I've made it so that the method raises a RuntimeError if the stream is not a subscription (or does not exist).